### PR TITLE
Docker Issue Fix

### PR DESCRIPTION
**Pull request by Saurav-D**
# Peer Review Information

Solves https://github.com/Unity-Technologies/datasetinsights/issues/112

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu18.04
+# FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu18.04
```
Comment (BlairLee) [inline Dockerfile]: Can we remove this comment?

**Comment by BlairLee** [comment]
Great job!

File: Dockerfile (hunk 1 above)
Comment (Saurav-D) [reply Dockerfile]: Sure, sorry forgot that.

**Review by BlairLee — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu18.04
+FROM nvidia/cuda:11.1-cudnn8-devel-ubuntu18.04
```
Comment (adason) [inline Dockerfile:1]: @Saurav-D Please revert this. This change will disable GPU support for pytorch as the package in pypi only supports CUDA 10 for pytorch 1.4.x.

File: pyproject.toml [hunk 3]
```diff
@@ -99,5 +99,5 @@ testpaths = [
 datasetinsights = "datasetinsights.__main__:entrypoint"
 
 [build-system]
-requires = ["poetry>=1.0.5"]
+requires = ["poetry>=1.0.10"]
```
Comment (adason) [inline pyproject.toml:102]: @Saurav-D This does not pin poetry version to 1.0.10.

File: Dockerfile (hunk 2 above)
Comment (Saurav-D) [reply Dockerfile:1]: @adason Is there any issue about this on pytorch? I launched train pipeline using this and it ran successfully using GPUs.

File: pyproject.toml (hunk 3 above)
Comment (Saurav-D) [reply pyproject.toml:102]: Yes. It's working without pinning poetry version.

File: Dockerfile (hunk 2 above)
Comment (Saurav-D) [reply Dockerfile:1]: Also, pip install downgrades cuda version to 10.1 as can be seen in the following:
```
# python
Python 3.7.5 (default, Nov  7 2019, 10:50:52) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> print(torch.version.cuda)
10.1
```

I ran the above inside the docker image of the latest build.

**Pull request merged**